nano/node/bootstrap/bootstrap.cpp

```diff
@@ -455,6 +452,14 @@ void nano::bootstrap_attempt::populate_connections ()
 	}
 }
 
+void nano::bootstrap_attempt::start_populate_connections ()
+{
+	if (populate_connections_started.exchange (true))
```

Comment:
The condition should be negated I think

> Return value
    The value of the atomic variable before the call. 

https://en.cppreference.com/w/cpp/atomic/atomic/exchange

Reply:
yep, i gave a bad snippet in the review 😬